rocAL/include/readers/image/image_reader.h

```diff
@@ -166,6 +174,26 @@ struct ImageRecordIOHeader {
                            */
 };
 
+struct NumpyHeaderData {
+   public:
+    std::vector<unsigned> array_shape;
+    RocalTensorDataType type_info;
+    bool fortran_order = false;
+    int64_t data_offset = 0;
+
+    RocalTensorDataType type() const { return type_info; }
+
+    size_t size() const {
+        size_t num_elements = 1;
+        for (const auto &dim : array_shape)
+            num_elements *= dim;
+        return num_elements;
+    };
+
+    size_t nbytes() const { return tensor_data_size(type_info) * size(); }
```

Comment:
Follow the same naming. Please change this to num_bytes similar to num_elements.